docs/en/setup/service-agent/java-agent/TLS.md

```diff
@@ -17,10 +17,23 @@ Only support **no mutual auth**.
 ## Open and config TLS
 
 ### Agent config
-- Place `ca.crt` into `/ca` folder in agent package. Notice, `/ca` is not created in distribution, please create it by yourself.
-
-- Agent open TLS automatically after the `/ca/ca.crt` file detected.
+- Agent open TLS automatically after the `ca.crt`(by default `/ca` folder in agent package) file detected.
 - TLS with no CA mode could be activated by this setting.
 ```
-agent.force_tls=${SW_AGENT_FORCE_TLS:false}
+agent.force_tls=${SW_AGENT_FORCE_TLS:true}
+```
+
+## Enable mutual TLS
+
+- Sharing gRPC server must be started with enabled mTLS. More details see `receiver-sharing-server` section in `application.yaml`. Please refer to [gRPC SSL](../../backend/grpc-ssl.md)  
```

Comment:
```suggestion
- Sharing gRPC server must be started with mTLS enabled. More details can be found in `receiver-sharing-server` section in `application.yaml`. Please refer to [gRPC SSL](../../backend/grpc-ssl.md)  
```

Reply:
The gRPC SSL refers to the main repo. So we cannot use a relative path. Could we use this URL, https://skywalking.apache.org/docs/main/latest/en/setup/backend/grpc-security/

Reply:
Please don't. This is a dev doc(notice `latest`). We never will have a static path for backend document because it changes with versions, including the URL. That is why I recommend for `key words` to search document. 

@Jtrust is going to add doc search for every versions of documents. Then it is more friendly and useful than right now.